components/widgets/api/src/main/resources/ApplicationResources.properties

```diff
@@ -67,3 +67,4 @@ phenotips.widgets.vocabularyBrowser.loading.error=Failed to retrieve data:
 phenotips.widgets.vocabularyBrowser.expand.error=Failed to retrieve data: 
 
 phenotips.widgets.detailsDialogGroup.termDeleteConfirmation=Are you sure you want to deselect this item? This will delete all associated data
+phenotips.widgets.detailsDialog.datePicker.pasteButton.title=Use last entered date
```

Comment:
[The convention](http://dev.xwiki.org/xwiki/bin/Community/L10N+Conventions#HRulesforstandardformelements) is to name this `.tooltip`.